astropy/io/fits/hdu/hdulist.py

```diff
@@ -750,18 +750,18 @@ def info(self, output=None):
         else:
             name = self._file.name
 
-        results = ['Filename: %s' % name,
+        results = ['Filename: {}'.format(name),
                    'No.    Name         Type      Cards   Dimensions   Format']
 
-        format = '%-3d  %-10s  %-11s  %5d   %-10s   %s   %s'
+        format = '{:3d}  {:10}  {:11}  {:5d}   {}   {}   {}'
```

Comment:
it gets an empty tuple as an input in the test case, and was very unhappy about having ``{:10}`` in, so one other point to take chances.